src/__swaps__/screens/Swap/components/AnimatedSwapCoinIcon.tsx

```diff
@@ -39,13 +40,13 @@ export const AmimatedSwapCoinIcon = React.memo(function FeedCoinIcon({
 }) {
   const { isDarkMode, colors } = useTheme();
 
-  const imageLoadingError = useSharedValue(false);
+  const didErrorForUniqueId = useSharedValue<string | undefined>(undefined);
 
   const animatedIconSource = useAnimatedProps(() => {
     return {
       source: {
         ...DEFAULT_FASTER_IMAGE_CONFIG,
-        borderRadius: (small ? 16 : large ? 36 : 32) / 2,
```

Comment:
This is the cause of https://linear.app/rainbow/issue/APP-1534/some-images-displayed-as-rectangles-on-android